src/components/MainDisplay.tsx

```diff
@@ -28,7 +28,7 @@ function MainDisplay(props: {
         </div>
         {/* TODO: Add respective image of car from databse */}
 
-        <img className="max-w-xs mx-auto" src={Mercedes} />
+        <img className="max-w-xs mx-auto" src={car.image} />
```

Comment:
We could maybe add the same alt here, and also the todo above this point can be removed.